src/NuGet.Clients/PackageManagement.UI/Feeds/MultiSourcePackageFeed.cs

```diff
@@ -196,11 +196,16 @@ private static LoadingStatus GetLoadingStatus(TaskStatus taskStatus)
             }
             else
             {
+                var items = nonEmptyResults.Select(r => r.Items).ToArray();
+
                 var indexer = new RelevanceSearchResultsIndexer();
                 var aggregator = new SearchResultsAggregator(indexer);
                 var aggregatedItems = await aggregator.AggregateAsync(
-                    searchText, nonEmptyResults.Select(r => r.Items).ToArray());
+                    searchText, items);
+
                 result = SearchResult.FromItems(aggregatedItems.ToArray());
+                // set correct count of unmerged items
+                result.RawItemsCount = items.Aggregate(0, (r, next) => r + next.Count);
```

Comment:
Now that I see the other changes, I think this is fine.

---
In reply to: [58757519](https://github.com/NuGet/NuGet.Client/pull/459#discussion_r58757519) [](ancestors = 58757519)